### PR TITLE
ffgambit's name card

### PR DESCRIPTION
**Pull request by ffgambit**
Added my card to the index.html file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -136,6 +136,37 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+        <!-- ________ ffGambit card START ________  -->
+        <div class="card">
+          <p class="name">ffGambit</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="www.twitter.com" target="_blank"></a>
```
Comment (Prajwalprakash3722) [inline index.html:144]: You haven't added your profile link to Twitter.

**Review by Prajwalprakash3722 — COMMENTED** [review]
Thank you for your contribution 🚀You haven't added your profile link to the Twitter tag, if you want to update it,surely do it 👍🏼

File: index.html (hunk 1 above)
Comment (ffgambit) [reply index.html:144]: I'm fine without. Do I need to go create an account for this?

Comment (Prajwalprakash3722) [reply index.html:144]: No need to create a Twitter account, Thank you for your contribution.🚀Best of luck in open source. 🎉

**Comment by Prajwalprakash3722** [comment]
Congratulations your PR has been merged 🚀best of luck in open sourcing 🎉

**Pull request merged**